build.gradle

```diff
@@ -361,6 +361,7 @@ javadoc.options {
 
 clean {
   delete 'tmp_gh-pages/'
+  delete 'tmp_docs/'
```

Comment:
Just hope it is not accidental.

Reply:
In https://github.com/googleapis/gax-java/pull/301 this directory was added as a temporary directory, and added to .gitignore, but it wasn't added to the clean task.